src/components/TransactionPanel/TransactionPanel.js

```diff
@@ -334,17 +336,19 @@ export class TransactionPanelComponent extends Component {
               onShowMoreMessages={() => onShowMoreMessages(currentTransaction.id)}
               totalMessagePages={totalMessagePages}
             />
+            {showSendMessageForm ? (
```

Comment:
This is great. 👍 

Maybe there could be an info text that states why you can't send messages atm.